metaflow/metaflow_config.py

```diff
@@ -125,6 +125,13 @@ def from_conf(name, default=None):
 # sandbox.
 SFN_STATE_MACHINE_PREFIX = from_conf("METAFLOW_SFN_STATE_MACHINE_PREFIX")
 
+###
+# AWS Lambda configuration
+###
+LAMBDA_ROLE_ARN = from_conf("METAFLOW_LAMBDA_ROLE_ARN")
+LAMBDA_IMAGE_URI = from_conf("METAFLOW_LAMBDA_IMAGE_URI")
+LAMBDA_THROTTLE_RETRIES = int(from_conf("METAFLOW_LAMBDA_THROTTLE_RETRIES", 20))
```

Comment:
What is the utility of this variable? I see that it's used to specify `max-attempts` for the lambda invocation. We should rely on the value obtained from `@retry` instead.

Reply:
This parameter controls retries in case we get throttled by AWS due to Lambda parallel execution limits. I assume users will rarely need to touch this, but it is still nice to have an option. `@retry` is still respected and controls retries in case of step failure.

Reply:
Ah got it! Can we make a note in the code base about this behavior so that it doesn't throw off future committers?